src/Transport/Topology/MetaModel/NamespaceBundleConfigurations.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace NServiceBus.AzureServiceBus.Topology.MetaModel
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    class NamespaceBundleConfigurations : IEnumerable<NamespaceBundleInfo>
+    {
+        List<NamespaceBundleInfo> namespaceBundles;
+
+        public NamespaceBundleConfigurations()
+        {
+            namespaceBundles = new List<NamespaceBundleInfo>();
+        }
+
+        public void Add(string namespaceAlias, int numberOfTopicsInBundle)
+        {
+            var bundleInfo = namespaceBundles.SingleOrDefault(x => x.Alias.Equals(namespaceAlias, StringComparison.OrdinalIgnoreCase));
+            if (bundleInfo != null && bundleInfo.NumberOfTopicsInBundle < numberOfTopicsInBundle)
+            {
+                throw new Exception($"Duplicate namespace alias `{namespaceAlias}` was added to the {nameof(NamespaceBundleConfigurations)}. Namespace aliases should be unique.");
+            }
+
+            namespaceBundles.Add(new NamespaceBundleInfo(namespaceAlias, numberOfTopicsInBundle));
+        }
+
+        public int GetNumberOfTopicInBundle(string namespaceAlias)
+        {
+            var selected = namespaceBundles.SingleOrDefault(x => x.Alias.Equals(namespaceAlias, StringComparison.OrdinalIgnoreCase));
+            return selected?.NumberOfTopicsInBundle ?? 0;
```

Comment:
It's ok to return 0 because we'll [take the maximum](https://github.com/Particular/NServiceBus.AzureServiceBus/blob/b58165d61aa661e2018c0dd3dbeb31913e775fc9/src/Transport/Topology/Topologies/ForwardingTopologySectionManager.cs#L280) between what is returned from this method and whatever is configured on the endopint.